apps/ledger-live-desktop/src/renderer/components/SignMessageConfirm/index.tsx

```diff
@@ -91,16 +106,22 @@ const SignMessageConfirm = ({ device, account, parentAccount, signMessageRequest
     }
   }
 
+  const contractAddress = (fields?.find(f => f.label === "Token") as { value: string })?.value;
```

Comment:
You shouldn't cast it 

```suggestion
 const contractAddress = fields?.find(f => f.label === "Token" && typeof f.value === 'string')?.value;
```